packages/paste-libraries/use-clipboard/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@twilio-paste/use-clipboard-library",
+  "version": "0.2.1",
+  "category": "library",
+  "status": "production",
+  "description": "A library to copy text to clipboard",
```

Comment:
Could this reference the library that this package exports? Essentially this is a wrapper around `use-clipboard-copy` which we should make clear. [The description for our reakit package](https://github.com/twilio-labs/paste/blob/main/packages/paste-libraries/reakit/package.json#L6) has an example of this.

Reply:
it feels like one goal of these Paste libraries is to hide the implementation detail e.g. are we using reakit, or in the future can we switch to some other library for tooltips? i followed the other library descriptions' naming patterns, imo the reakit naming increases coupling to a particular library vs being broken out into different packages (Disclosure, Menu, Popover, Portal, Tab, Tooltip)

Reply:
Suggestion: `A library that allows the user to copy text with the click of a button.`

Reply:
actually thinking about this more, maybe this should be the description of the component and not the library. the library will allow you to attach copy functionality to any kind of action (e.g. upon updating sharing permissions with a user, will copy the URL in the success callback)